packages/file_selector/file_selector_platform_interface/lib/src/method_channel/method_channel_file_selector.dart

```diff
@@ -79,18 +76,32 @@ class MethodChannelFileSelector extends FileSelectorPlatform {
     );
   }
 
-  /// Gets a directory path from a dialog
   @override
   Future<String?> getDirectoryPath({
     String? initialDirectory,
     String? confirmButtonText,
   }) async {
-    return _channel.invokeMethod<String>(
+    final List<String>? pathList = await _channel.invokeListMethod<String>(
       'getDirectoryPath',
       <String, dynamic>{
         'initialDirectory': initialDirectory,
         'confirmButtonText': confirmButtonText,
       },
     );
+    return pathList?.first;
+  }
+
+  @override
+  Future<List<String>> getDirectoryPaths(
+      {String? initialDirectory, String? confirmButtonText}) async {
+    final List<String>? pathList = await _channel.invokeListMethod<String>(
+      'getDirectoryPath',
```

Comment:
And this will need to be a different method name for the same reason.

Reply:
and keep `getDirectoryPaths` here?